example/cumulus-tf/terraform.tfvars.example

```diff
@@ -126,3 +126,11 @@ additional_log_groups_to_elk = {
   "LogDescriptor" = "log_group_name"
 }
 */
+
+# Optional, configure if needed. Adds policies to SNS report topics.
+#  If a Principal is specified as just an AWS account ID rather than an ARN,
+#    AWS silently converts it to the ARN for the root user,
+#    causing future terraform plans to differ. To avoid this problem,
+#    specify the full ARN.
+# add_report_topic_policy = true
+# metrics_account_id = ["arn:aws:iam::12345678:user/FederatedUser"]
```

Comment:
Suggest to have a more generic name like `report_sns_topic_subscriber_arns`

Reply:
Please check when the value is `[]`, the subscription from the owner account still works.
Also verify it works for multiple subscriber arns.

Reply:
`[]` will now work, I updated the default value for the variable to be `[*]` https://github.com/nasa/cumulus/pull/3482/commits/c51dc64f18f498676ca89d40b1b71ad5e84fd33d#diff-923ccc457843eaf76dcda5594381ce2124f7cf184108342978c10da459c88aecR488 so if a user supplies `report_sns_topic_subscriber_arns = []` in their `terraform.tfvars`, the deployment will still succeed and use the default